### PR TITLE
Move upate_alias to manager

### PR DESCRIPTION
**Pull request by kimpepper**
Marked as deprecated, as there are quite a few usages of this method, and I wanted to do an initial conversion.

### CODE REVIEW

**Comment by kimpepper** [comment]
Not entirely sure what the failure is. The tests all pass.

File: src/PathautoManager.php [hunk 1]
```diff
@@ -453,4 +458,51 @@ public function resetCaches() {
     $this->cleanStringCache = array();
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function updateEntity(EntityInterface $entity, $op, array $options = array()) {
```
Comment (Berdir) [inline src/PathautoManager.php]: Not sure about the name, we don't update the the entity, we update/create an alias for the entity? So maybe updateAlias($entity, $op) ? or updateEntityAlias()?

File: src/PathautoManager.php [hunk 2]
```diff
@@ -453,4 +458,51 @@ public function resetCaches() {
...
+    }
+
+    $result = $this->createAlias(
+      $type, $op, $entity->getSystemPath(), array($type => $entity), $bundle, $options['language']);
+
+    if ($type == 'taxonomy_term' && empty($options['is_child'])) {
+      // For all children generate new aliases.
+      $options['is_child'] = TRUE;
+      unset($options['language']);
+      foreach (taxonomy_get_tree($entity->getVocabularyId(), $entity->id(), NULL, TRUE) as $subterm) {
```
Comment (Berdir) [inline src/PathautoManager.php]: Do we want hide taxonomy_get_tree() in a helper method, to ease unit testing later on?

**Comment by Berdir** [comment]
Looks fine, we can always improve this by splitting up some to protected methods or move logic to plugins, once we have them.

**Pull request merged**